examples/tanstack-start/app/routes/__root.tsx

```
@@ -8,7 +8,7 @@ import {
 } from '@tanstack/react-router';
 import { TanStackRouterDevtools } from '@tanstack/router-devtools';
 import { Body, Head, Html, Meta, Scripts } from '@tanstack/start';
-import { createServerSideHelpers } from '@trpc/react-query/server';
+import { TRPCOptionsProxy } from '@trpc/tanstack-react-query';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)